pyproject.toml

```diff
@@ -1,10 +1,15 @@
 [tool.poetry]
 name = "bqemulatormanager"
-version = "0.1.8"
-description = ""
-authors = ["gyuta <kuroshiba0408@gmail.com>"]
+version = "0.0.0"  # using poetry-dynamic-versioning
+description = "bqemulatormanager is a wrapper of bigquery-emulator which provides us BigQuery mock working in local machine."
+authors = ["gyuta <kuroshiba0408@gmail.com>", "M3, Inc."]
```

Comment:
maybe we should add these lines on here
https://github.com/m3dev/gokart/blob/master/pyproject.toml#L8-L9